lrc/cardano-sl-lrc.cabal

```diff
@@ -83,3 +84,26 @@ library
 
   build-tools: cpphs >= 1.19
   ghc-options: -pgmP cpphs -optP --cpp
+
+test-suite cardano-sl-lrc-test
```

Comment:
Probably worth working towards some of the great conventions I've been talking about. In that light, that test executable should probably just be called "test".